src/index.html

```diff
@@ -11,7 +11,167 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form 
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application" 
+      method="POST" 
+      autocomplete="off">
+      <fieldset>
+        <legend>Personal information</legend>
+        <div>
+          <label class="form-field">
+            Surname: <input type="text" name="surname">
+          </label>
+        </div>
+        <div>
+          <label class="form-field">
+            Name: <input type="text" name="name">
+          </label>
+        </div>
+        <div>
+          <label class="form-field">
+            How old are You? 
+            <input 
+              type="number" 
+              name="age" 
+              value="12" 
+              min="1" 
+              max="100"
+            >
+          </label>
+        </div>
+        <div>
+          <label class="form-field">
+            Full date of birth: 
+            <input 
+              type="date" 
+              name="birthday" 
+              placeholder="mm / dd / yyyy"
+            >
+          </label>
+        </div>
+        <div>
+          <label class="form-field">
+            I accept the term of the agreement  
+            <input 
+              type="checkbox" 
+              name="agreement"
+              id="agreement" 
+              required
+            >
+          </label>
+        </div>
+      </fieldset>
+  
+      <fieldset>
+        <legend>Registration</legend>     
+        <div>
+          <label class="form-field">
+            E-mail: 
+            <input 
+              type="email" 
+              name="email" 
+              placeholder="email@example.com" 
+              required
+            >
+          </label>
+        </div>
+        <div>
+          <label class="form-field">
+            Password: 
+            <input 
+                type="password" 
+                name="password" 
+                minlength="5" 
+                maxlength="32"
+                required
+            >
```

Comment:
```suggestion
            <input 
              type="password" 
              name="password" 
              minlength="5" 
              maxlength="32"
              required
            >
```
There should be only 2 spaces of indentation